docs/api-reference/cloud-cli.md

```diff
@@ -298,4 +301,50 @@ It is meant to be used with [`database restore`](#npx-dbos-cloud-db-restore) dur
 - `-d, --database <string>` The name of the new database instance for this application.
 - `-p, --previous-version [number]`: The ID of a previous version of this application. If this is supplied, redeploy that version instead of deploying from the application directory. During restoration, we recommend deploying to the version active at the timestamp to which you recovered. You can list previous versions and their IDs and timestamps with the [versions command](#npx-dbos-cloud-app-versions).
 
+---
+
+## Organization Management Commands
+
+### `npx dbos-cloud org list`
+
+**Description:**
+List users in your organization
+
+**Parameters:**
+- `--json`: Emit JSON output
+
+---
+
+### `npx dbos-cloud org invite`
+
+**Description:**
+Generate an organization secret with which to invite another user into your organization. Organization secrets are single-use and expire after 24 hours.
```

Comment:
We should note that the secret will be consumed even in the presence of an error (e.g., you used the secret to join a different organization than the one that generated the secret)

Reply:
I didn't realize that, we should fix that behavior. Very confusing to lose your invite to a typo.